src/TextArea/TextArea.less

```diff
@@ -19,34 +18,36 @@
   top: @size_xs;
   width: 100%;
   color: @neutral_dark_gray;
+}
 
-  .TextArea--label {
-    left: @size_s;
-    opacity: 1;
-  }
+.TextArea--label {
+  left: @size_s;
+  opacity: 1;
+}
 
-  .TextArea--required {
-    position: absolute;
-    right: @size_l;
-  }
+.TextArea--required {
+  position: absolute;
+  right: @size_l;
+}
 
-  .TextArea--error {
-    color: @alert_red;
-    position: absolute;
-    right: @size_l;
-  }
+.TextArea--error {
+  color: @alert_red;
+  position: absolute;
+  right: @size_l;
 }
 
 .TextArea--input {
   .text--medium;
   display: block;
   width: 100%;
   .border--s(transparent);
+  border-top-width: @size_l;
```

Comment:
Regarding why I've used a transparent border to position the `TextArea` input below the label instead of padding or a margin:

- Padding doesn't work because text is still visible in the padding region.
- Margins don't work because the underlying `textarea` element overwrites margins (and height) when the expander is used to change the height of the element.